core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/SummaryTableDocFormatter.java

```diff
@@ -7,8 +7,8 @@
 
 class SummaryTableDocFormatter implements DocFormatter {
     private static final String TABLE_CLOSING_TAG = "\n|===";
-    private static final String TABLE_ROW_FORMAT = "\n\n|<<%s, %s>>\n|%s %s\n|%s\n| %s";
-    private static final String TABLE_HEADER_FORMAT = "== Summary\n%s|===\n|Configuration property|Type|Default|Lifecycle";
+    private static final String TABLE_ROW_FORMAT = "\n\n|<<%s, %s>>\n\n%s|%s %s\n|%s\n| %s";
+    private static final String TABLE_HEADER_FORMAT = "== Summary\n%s\n[cols=\"50,10,10,5\"]\n|===\n|Configuration property|Type|Default|Lifecycle";
```

Comment:
Is it normal that the sum is not 100%?

Reply:
Yeah, OCD does not apply to percentage sums in HTML columns :)